gitpod-io/gitpod
# Update GitLab logo

## Description

Logo update for GitLab, see [relevant blog post](https://about.gitlab.com/blog/2022/04/27/devops-is-at-the-center-of-gitlab/). 🍊

See also relevant changes in https://github.com/gitpod-io/website/pull/2001. 🪂 

## How to test
1. Open `/workspaces` as anonymous user and notice the new GitLab logo on the providers list.
2. Adittionally, try creating a new project and notice the new GitLab logo when selecting a different provider.

You can safely ignore the provider order in the screenshots below (production vs. preview environment) which is deliberately different.

## Screenshots

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-05-09 at 11 45 26 AM" src="https://user-images.githubusercontent.com/120486/167374231-df51f778-1ff2-4bbc-8459-c533e330a73f.png"> | <img width="1440" alt="Screenshot 2022-05-09 at 11 45 33 AM" src="https://user-images.githubusercontent.com/120486/167374140-088da5da-9719-4d5f-b0f4-254664f45000.png"> |
| <img width="1440" alt="Screenshot 2022-05-09 at 11 45 16 AM" src="https://user-images.githubusercontent.com/120486/167374609-4815a9cd-10a4-461c-ae6e-69a6d735f929.png"> | <img width="1440" alt="Screenshot 2022-05-09 at 11 45 46 AM" src="https://user-images.githubusercontent.com/120486/167374632-3bcbae30-44e7-4384-9fba-960acb55f2f3.png"> |

## Release Notes

```release-note
Update GitLab logo
```
